ui/app/insights/src/index.ts

```diff
@@ -54,13 +53,6 @@ fetch(configURL).then(async (result) => {
         }
     }
 
-    // Add config prefix if defined (used in dev)
```

Comment:
Any reason you have removed this?

Reply:
I don't exactly know how translations work but I assumed that there wouldn't be a need to use CONFIG_URL_PREFIX now that the managerConfig is served by the backend. Do you think that's not the case?

Reply:
I've reverted that just in case.